addons/mail/static/src/core/web/messaging_menu_quick_search.xml

```diff
@@ -0,0 +1,13 @@
+<t t-name="mail.MessagingMenuQuickSearch">
+    <div class="position-relative w-50 m-1">
+        <input
+        class="form-control px-1 py-0"
+        placeholder="Quick search..."
+        t-model="store.discuss.searchTerm"
+        t-ref="autofocus"
+        type="text"
+        />
```

Comment:
Avoid attempting to multi-line in XML.

We know XML sucks, and the consensus is to inline almost all the time.

There are few cases were multi-line is fine. Notably `t-att-class="{}"`.
In the eventuality you multi-line in XML, prefer the following indentation:
```xml
<div class="position-relative w-50 m-1">
    <input
        class="form-control px-1 py-0"
        placeholder="Quick search..."
        t-model="store.discuss.searchTerm"
        t-ref="autofocus"
        type="text"
    />
    <button
        class="btn btn-link text-muted p-1 position-absolute top-50 end-0 translate-middle-y"
        t-on-click="props.onClose"
    >
        <i class="oi oi-close"/>
    </button>
</div>
```

While this looks good to spot details, the drawback of this formatting is that we lack macro-view of template.
Namely it's not obviously apparent (=without thinking at all) that this is just `div` with `input` and `button`.
We value macro-view of templates quite a lot more than details, so I'd strongly avoid multi-lining in XML unless `t-att-class="{}"` with at least 2 conditionals.